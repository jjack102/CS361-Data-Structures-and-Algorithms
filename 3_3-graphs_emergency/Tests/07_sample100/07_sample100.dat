0 99 0
0 84 15 0
84 73 9 0
73 70 1 0
70 39 13 0
39 16 20 1
16 78 16 0
78 61 4 0
61 99 5 0
26 7 11 0
16 55 18 1
87 46 15 0
66 39 8 0
18 46 1 0
21 47 13 0
24 83 14 0
96 17 7 1
68 50 11 0
99 54 6 0
69 7 1 0
79 83 14 1
46 73 18 1
63 77 19 1
2 30 15 0
40 80 10 0
37 8 10 0
36 24 14 1
6 54 7 1
63 24 20 0
0 36 5 1
23 77 15 0
71 20 11 1
94 39 10 1
22 46 8 0
45 43 4 1
53 76 4 1
26 51 1 1
49 89 17 0
29 3 5 0
71 21 6 0
84 8 8 0
30 20 6 0
32 36 3 0
74 47 2 0
61 27 11 0
35 36 19 1
4 7 8 1
56 50 10 1
98 31 14 1
74 49 4 1
72 53 9 0
53 74 5 1
94 82 12 1
77 66 12 0
3 61 11 1
74 33 2 0
33 59 15 0
70 8 4 0
58 83 3 1
87 94 12 1
19 91 15 0
1 28 2 0
41 51 14 0
11 90 10 0
47 37 11 1
39 20 15 0
90 44 16 0
46 37 12 0
26 23 11 0
3 30 14 0
37 29 11 0
15 70 4 0
40 37 6 0
30 99 15 0
20 17 3 0
45 92 5 1
8 98 19 0
72 2 10 0
49 35 12 0
92 97 11 0
72 83 8 1
69 63 5 0
28 15 16 1
0 6 1 0
90 55 19 0
88 27 19 0
84 4 14 0
98 77 7 0
41 80 20 1
76 14 11 0
98 21 10 0
58 6 3 0
71 82 4 1
60 20 11 0
17 4 16 1
17 89 8 0
88 38 18 1
4 68 18 1
53 4 1 0
74 96 10 0
11 63 4 1
23 96 20 1
19 73 2 0
25 60 8 0
54 1 10 0
26 4 8 0
74 84 16 0
62 45 10 0
18 23 14 1
17 35 17 1
27 87 5 0
25 67 16 0
25 29 20 1
87 31 18 0
86 62 14 0
27 49 11 0
77 2 16 0
97 96 6 1
28 22 12 1
8 34 10 1
23 35 6 1
38 49 4 1
71 0 11 1
46 90 3 0
23 64 17 0
77 3 9 0
51 2 7 0
22 76 7 0
84 10 20 1
7 22 18 1
24 86 12 0
71 53 2 0
89 69 1 0
49 69 15 0
91 9 16 1
3 67 17 0
44 41 17 0
41 15 14 0
10 17 4 1
79 87 15 1
72 30 7 0
42 46 6 1
58 26 18 0
8 18 11 1
0 32 16 0
4 23 9 0
37 45 19 0
92 86 5 0
14 36 2 0
96 69 15 1
55 92 20 0
10 23 10 0
12 4 2 0
38 22 15 1
73 69 20 1
10 97 9 0
99 52 11 0
43 10 11 0
30 52 18 1
28 82 4 1
36 84 12 0
5 74 4 0
30 50 6 0
12 96 12 0
38 60 11 0
66 93 8 0
53 28 16 1
68 17 18 1
32 59 12 0
40 57 14 0
28 2 20 0
54 80 1 0
6 98 8 0
90 88 9 1
2 18 15 0
70 38 9 0
2 27 19 0
64 84 7 1
81 67 12 0
37 99 15 0
12 42 4 0
27 81 16 0
60 12 3 1
40 52 8 0
69 68 16 1
54 24 17 0
0 60 6 1
74 90 16 1
0 76 11 0
60 70 9 1
66 63 19 0
97 31 10 1
99 27 13 0
60 80 13 0
46 49 5 0
79 59 10 0
44 59 13 1
40 4 11 0
71 32 9 0
8 40 1 0
52 14 2 1
16 53 10 0
26 42 5 1
43 26 4 0
70 4 12 0
96 37 11 0
19 45 13 0
20 18 18 0
24 80 7 0
51 71 1 0
29 82 2 0
30 73 3 0
41 22 14 1
29 64 13 0
6 89 2 1
57 58 17 0
3 73 1 0
63 29 1 1
68 59 2 0
55 38 11 1
15 3 8 1
75 86 14 0
1 95 12 0
56 25 17 0
27 7 6 0
48 57 12 0
42 30 19 0
82 35 16 1
52 12 6 0
0 10 6 0
1 9 20 0
21 57 1 0
82 84 20 0
86 2 16 0
95 94 9 0
41 10 14 0
97 16 18 0
96 98 20 1
64 53 3 0
76 91 10 0
12 24 17 0
0 54 18 0
55 83 7 1
42 59 20 1
47 94 1 1
70 0 11 0
3 43 11 0
57 37 12 0
79 44 17 0
40 75 12 1
90 81 5 1
85 87 5 0
49 34 16 0
38 13 3 0
87 14 3 0
57 51 17 0
90 57 5 0
6 45 7 0
92 0 19 0
57 39 16 0
94 79 14 0
40 49 2 1
22 49 1 1
86 1 19 1
45 26 12 0
76 35 6 0
59 14 18 0
23 9 1 0
48 13 5 0
74 87 15 0
27 83 19 0
38 17 18 0
41 96 17 0
15 33 14 0
96 72 13 1
25 42 9 0
51 65 19 0
91 25 7 0
14 30 7 1
26 60 20 0
8 55 6 1
95 68 9 0
69 70 6 0
29 69 12 0
56 62 4 0
43 47 20 0
17 91 18 0
71 42 17 0
34 74 5 0
59 77 5 0
99 26 3 1
7 77 20 1
38 16 7 0
32 85 11 0
36 53 10 0
8 17 15 0
74 43 6 0
6 63 13 0
77 90 5 1
47 72 4 0
89 65 11 0
63 60 2 0
59 97 5 0
86 71 17 1
27 85 11 1
97 34 10 1
2 99 18 1
72 38 13 0
52 80 6 0
9 82 1 0
95 38 5 0
72 43 13 0
10 61 3 0
60 74 5 0
73 71 14 1
23 59 6 0
42 50 16 0
96 91 7 0
85 10 20 0
78 39 12 0
9 51 8 0
83 37 3 1
93 7 14 0
69 2 12 0
41 65 11 1
93 13 6 0
60 3 12 0
26 72 5 0
29 99 20 0
43 53 12 1
10 86 2 1
3 58 8 1
29 40 20 0
69 8 20 0
32 30 8 0
11 62 9 0
96 46 16 0
99 89 3 0
6 21 19 0
44 47 13 1
38 37 19 0
61 98 3 0
92 39 20 0
70 22 15 0
39 65 1 0
75 1 15 0
60 33 6 0
71 81 10 1
93 3 19 1
11 4 4 0
2 68 11 0
49 8 13 0
89 93 15 0
56 27 20 1
82 69 2 0
25 75 1 0
8 47 1 0
97 99 13 1
13 72 3 0
1 16 15 0
54 61 4 0
1 25 16 1
80 5 3 1
50 49 6 0
65 53 17 1
18 67 20 0
61 7 12 0
60 17 17 0
87 43 11 0
86 64 19 0
18 47 11 0
12 67 15 0
46 77 15 0
39 47 8 1
60 47 5 0
71 54 3 0
12 44 11 0
53 29 1 0
49 23 18 0
96 10 7 0
30 57 6 0
80 17 12 1
52 18 9 1
82 22 15 0
65 46 8 0
89 64 15 0
75 80 5 0
63 20 14 0
25 24 8 1
19 48 19 1
30 13 2 1
4 42 16 1
24 58 2 0
32 20 10 0
20 14 5 0
29 52 12 0
32 81 13 0
23 66 19 1
3 88 3 1
64 44 12 0
4 39 18 0
24 68 13 0
11 2 8 0
44 31 4 0
50 74 6 0
96 54 11 0
69 57 14 0
63 93 15 0
67 95 14 0
13 87 5 0
10 95 11 0
92 94 20 0
76 15 3 0
12 32 16 1
55 76 15 0
28 60 19 0
67 53 17 0
28 78 10 0
55 72 8 1
61 57 11 1
98 9 15 1
66 27 4 0
12 33 19 0
75 78 15 1
73 37 14 0
73 65 3 0
31 1 19 0
80 95 19 1
85 35 4 0
23 14 1 0
47 90 4 0
19 75 5 1
20 74 2 0
99 34 16 0
83 98 7 0
75 30 6 1
43 50 11 1
8 68 12 1
39 33 19 0
33 22 5 0
34 65 9 0
53 40 19 0
34 0 13 1
82 53 3 0
28 54 14 0
79 27 9 0
88 62 14 1
77 32 10 0
94 37 7 1
77 37 9 1
39 85 15 0
58 78 5 1
18 91 15 0
0 4 5 1
9 17 6 0
24 62 20 0
9 58 20 0
62 43 4 0
63 42 7 0
49 42 3 0
57 99 20 0
22 19 9 0
94 16 4 0
56 23 1 0
5 95 7 1
73 78 14 1
7 73 6 0
35 16 9 0
78 51 5 1
70 92 16 0
46 33 6 0
35 95 15 0
28 57 5 0
85 37 1 0
56 7 15 0
56 3 2 0
75 12 6 0
26 33 20 1
7 87 4 0
48 32 2 0
17 55 12 0
5 52 14 1
71 47 4 0
64 1 9 1
50 34 19 0
46 48 2 0
19 81 11 1
43 99 13 0
38 81 18 0
36 46 9 1
40 92 1 0
4 80 11 0
1 63 4 0
19 49 19 0
95 88 1 0
41 28 1 0
43 28 7 0
1 14 20 0
61 97 8 0
14 51 2 0
78 2 16 0
97 70 2 0
47 51 4 0
38 51 4 1
31 39 19 0
67 50 5 0
89 14 13 0
38 46 17 1
67 31 16 0
26 70 8 0
88 77 18 0
98 49 11 0
74 64 2 0
18 50 6 0
75 65 2 0
1 13 7 1
69 32 14 0
5 36 14 1
34 46 2 1
2 24 10 0
92 21 8 0
56 70 19 1
92 48 13 0
31 16 1 0
32 19 13 0
45 82 8 0
69 20 15 0
76 16 6 1
71 3 20 0
82 72 10 0
69 51 13 0
56 74 10 0
39 42 16 0
20 41 7 1
67 64 12 0
15 75 16 0
86 7 14 0
23 17 4 0
75 7 12 0
37 20 17 0
89 58 3 0
31 21 18 1
24 90 14 0
27 93 17 1
93 36 5 0
19 84 6 1
78 91 14 0
77 97 15 1
7 92 11 0
72 94 5 0
99 48 1 0
11 26 5 0
55 0 10 0
47 97 3 1
10 91 3 0
36 42 9 1
97 89 15 0
34 76 14 0
34 27 8 0
12 54 13 0
63 43 8 0
97 41 20 1
40 5 17 0
33 65 9 0
99 85 20 1
13 62 13 0
11 68 16 1
15 10 6 1
65 25 17 0
2 54 18 0
19 66 19 0
98 16 8 0
55 33 3 1
93 90 13 0
12 57 2 1
44 76 5 1
0 93 15 0
17 93 1 0
98 2 2 1
58 49 2 1
87 71 5 0
64 99 13 1
91 79 2 0
56 63 17 0
63 76 17 0
92 85 1 0
98 37 3 0
75 11 1 1
95 71 20 1
65 8 18 1
40 15 10 0
10 50 14 0
17 50 10 0
73 81 18 0
49 65 7 0
12 10 16 0
26 68 4 0
74 89 15 0
44 4 17 0
54 89 7 1
41 53 17 0
26 17 20 0
16 73 6 0
40 55 8 1
21 5 17 0
15 13 10 0
73 77 11 0
64 90 13 0
27 89 20 0
71 28 15 0
54 83 13 0
91 80 1 0
21 48 1 0
21 28 5 1
37 62 6 0
81 63 1 0
89 72 2 1
4 10 18 0
29 19 16 1
75 2 16 0
48 91 2 1
92 28 9 1
81 15 11 1
56 87 4 0
98 47 15 0
20 2 20 0
64 93 11 0
2 37 14 0
21 39 7 0
52 2 19 0
51 43 2 0
7 55 10 0
5 37 4 0
74 81 14 0
39 75 4 0
24 84 3 1
3 57 10 0
99 59 9 1
45 50 16 1
59 17 15 0
52 75 14 0
10 6 16 1
13 84 18 1
45 65 7 0
86 99 19 0
0 38 19 0
43 8 17 1
93 48 19 0
45 20 15 0
81 36 10 0
95 6 5 0
20 50 15 0
63 72 15 0
9 60 7 0
22 20 11 1
77 26 19 1
96 40 10 0
32 61 13 0
24 77 18 0
78 62 14 1
86 46 4 1
18 53 12 0
72 65 13 0
11 44 11 0
15 27 14 1
87 38 17 0
97 17 12 0
64 45 4 0
9 48 9 0
6 33 7 0
25 84 6 1
95 54 8 0
14 48 14 0
77 1 6 0
94 49 10 0
36 64 20 0
29 18 4 0
5 96 5 0
45 21 13 0
72 85 14 0
60 21 8 0
54 76 15 0
60 91 4 0
60 75 9 1
39 38 12 0
28 85 16 0
77 30 4 0
65 82 17 1
26 10 14 0
3 51 20 0
95 15 13 0
72 49 8 0
44 39 14 1
64 21 3 0
15 60 20 1
75 76 11 0
4 36 10 1
3 95 17 0
26 31 7 1
81 65 10 0
47 82 12 1
64 76 8 0
65 29 19 0
58 43 3 0
39 62 17 0
29 67 6 0
88 73 5 0
6 53 5 0
65 22 18 0
10 52 3 1
65 36 8 1
35 98 15 1
53 35 2 0
32 17 2 0
11 97 20 0
98 70 6 0
41 38 19 1
84 87 9 1
77 13 20 0
54 17 12 1
39 12 17 0
22 85 6 0
86 88 3 0
59 35 11 0
9 21 1 0
40 26 8 0
9 27 10 0
62 61 9 0
32 74 16 0
58 31 19 0
96 51 5 0
21 80 9 0
39 50 13 1
67 36 4 0
58 42 12 0
22 44 20 0
82 33 4 1
31 91 13 0
55 23 19 0
70 64 5 0
51 46 9 0
79 65 11 0
7 12 5 0
39 18 1 1
95 87 20 0
21 68 7 0
19 12 6 0
18 65 19 0
58 37 4 0
1 68 14 0
79 75 8 1
12 61 7 0
61 95 18 0
5 16 9 0
32 70 14 0
73 12 6 0
35 33 17 0
76 27 12 0
3 23 6 0
66 24 14 0
91 86 2 0
89 9 1 0
3 99 7 0
98 4 9 0
67 78 16 0
32 79 17 0
91 93 6 0
74 48 7 0
61 45 6 0
75 93 10 0
67 1 12 0
34 85 5 1
82 37 15 0
8 48 2 1
13 99 19 0
60 89 18 0
1 11 18 0
46 68 2 1
89 78 13 0
99 32 7 0
33 84 3 0
71 8 3 1
77 39 10 1
9 87 3 0
72 28 8 1
28 98 17 0
32 15 12 1
0 45 19 0
4 54 20 0
22 97 18 0
80 74 18 0
81 3 1 0
0 14 9 0
87 23 12 1
75 17 4 0
62 84 20 0
56 8 12 0
32 28 19 1
36 16 7 1
61 65 19 0
14 46 5 0
91 88 5 0
69 47 20 0
39 43 14 0
96 71 7 0
17 77 9 0
27 69 10 0
92 66 3 0
93 67 16 0
83 7 10 0
3 91 15 1
68 87 9 0
20 4 7 0
4 33 11 0
21 18 12 0
79 5 19 0
33 70 9 0
94 54 16 0
69 85 18 1
50 65 19 0
56 19 3 0
95 83 19 0
89 87 9 0
6 85 13 0
41 61 7 0
77 9 6 0
66 53 7 0
82 39 20 1
45 57 5 0
84 39 18 0
76 13 3 1
23 32 4 0
33 18 11 0
68 74 11 0
57 70 11 1
8 15 11 1
7 1 8 0
77 84 17 0
63 78 15 1
85 29 3 1
83 20 15 1
20 6 6 1
56 79 15 1
76 32 13 0
44 0 19 0
18 85 7 1
51 28 19 0
7 11 16 1
43 14 12 0
51 76 17 0
50 36 6 0
57 33 6 0
56 77 12 0
92 42 1 0
95 7 15 0
13 51 19 1
86 89 4 0
36 31 1 0
75 83 13 1
63 88 20 0
56 14 7 0
16 58 15 1
51 36 19 0
54 98 6 0
43 19 7 0
49 82 3 1
65 2 12 1
42 77 6 0
14 84 14 0
41 54 1 0
33 69 13 0
57 97 20 0
15 26 7 0
82 2 17 0
0 83 17 1
7 2 1 0
65 67 14 0
91 82 20 1
69 34 17 1
60 35 6 0
18 1 18 1
41 4 15 1
70 83 1 0
53 80 4 0
80 68 2 1
6 40 6 0
92 73 7 1
16 20 10 1
66 81 9 0
39 58 11 1
16 61 14 0
92 76 11 0
96 62 10 0
66 0 4 0
92 38 6 1
46 31 20 0
47 38 9 0
56 59 16 1
71 69 18 0
72 87 15 0
89 90 7 0
20 80 13 0
48 36 8 0
41 92 6 0
7 42 3 1
80 25 4 1
94 53 3 0
97 26 8 0
58 41 16 0
64 4 17 0
24 11 11 1
49 47 8 0
5 90 3 0
66 64 12 0
82 30 14 0
15 18 3 0
39 64 9 0
44 49 13 1
57 60 12 0
54 81 16 0
24 49 11 0
13 47 10 1
67 62 18 1
94 24 5 1
4 31 11 1
1 26 6 0
21 19 1 0
90 4 5 0
50 96 8 0
93 26 19 1
82 60 6 1
76 41 13 0
71 33 7 1
54 77 14 0
66 78 7 0
78 79 11 1
93 71 5 1
21 97 2 0
54 74 13 0
24 76 14 0
94 66 2 0
80 96 17 0
63 25 17 1
38 19 9 0
19 14 8 0
13 42 9 1
0 67 19 1
71 18 11 1
97 49 17 0
98 74 14 0
98 91 17 1
97 19 16 1
79 39 1 1
83 59 14 1
87 98 6 0
37 10 8 0
96 75 11 0
41 46 14 0
9 93 16 0
13 32 6 0
33 10 13 0
53 33 20 0
50 94 6 0
85 95 8 0
11 64 18 1
87 47 10 0
76 74 1 1
36 41 19 1
92 18 14 0
8 33 18 0
64 69 3 1
27 30 10 1
1 91 1 0
17 6 3 0
62 19 17 1
53 99 3 1
67 16 14 0
16 54 11 0
15 44 16 0
28 61 14 1
35 15 15 0
86 51 11 0
84 65 19 1
33 38 20 0
56 26 12 1
24 9 15 0
7 94 13 0
21 32 19 1
41 89 19 0
38 57 1 0
85 19 14 0
18 94 15 0
47 23 9 1
74 24 6 1
51 60 7 0
14 7 9 1
86 90 13 0
69 93 19 0
84 37 11 0
14 50 7 0
62 87 18 1
