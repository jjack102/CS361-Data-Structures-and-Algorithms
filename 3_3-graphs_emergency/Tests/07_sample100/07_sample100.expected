Fastest time for car at 0 to the emergency at 99 is 9 minutes.
